packages/connect-web-test/src/cancellation.spec.ts

```diff
@@ -1,116 +0,0 @@
-// Copyright 2021-2024 The Connect Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-import type { CallOptions } from "@connectrpc/connect";
-import {
-  Code,
-  ConnectError,
-  createCallbackClient,
-  createPromiseClient,
-} from "@connectrpc/connect";
-import { describeTransports } from "./helpers/conformanceserver.js";
-import { TestService } from "./gen/connectrpc/conformance/v1/test_connect.js";
-
-describe("explicit cancellation with AbortController", function () {
```

Comment:
Do we test callback clients with the conformance test runner? If not, we need to keep coverage for this.